dependency_injection/services.php

```diff
@@ -47,6 +47,9 @@
     $parameters->set('env(APP_SECRET_FILE)', $projectDir . '/config/glpicrypt.key');
     $parameters->set('kernel.secret', env('default:glpi.default_secret:file:APP_SECRET_FILE'));
 
+    // Prevent low level errors (e.g. warning) to be converted to exception in dev environment
```

Comment:
This is because of a too high number of this errors ?

Reply:
No, it is because we used to use warnings in GLPI for non blocking errors, and I wanted to preserve this behaviour for the moment.